src/components/Company/CompanyName/CorrectNameOptions.vue

```diff
@@ -67,23 +80,28 @@ export default class CorrectNameOptions extends Vue {
   // local properties
   private displayedOptions: Array<CorrectNameOptionIF> = []
   private panel = null as number
-  private submit = false
+  private submitId = ''
+  private currentPanelId = ''
```

Comment:
Are these strings or numbers?
(PS You can initialize numbers to NaN if needed.)

Reply:
This has been refactored in next commit. The `id` was misleading, it is a type.